concordium-contracts-common/src/types.rs

```diff
@@ -1250,6 +1344,15 @@ impl fmt::Display for NewAttributeValueError {
 #[derive(Clone, Copy, Debug, Ord, PartialOrd, Eq, PartialEq)]
 pub struct AttributeTag(pub u8);
 
+#[cfg(feature = "concordium-quickcheck")]
+impl quickcheck::Arbitrary for AttributeTag {
+    fn arbitrary(g: &mut Gen) -> AttributeTag { AttributeTag(quickcheck::Arbitrary::arbitrary(g)) }
```

Comment:
Please document the discussions we had here about why we do not only generate tags 0-14.

Reply:
Done.